scripts/check_padding_mm.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torchtitan.datasets import build_mm_data_loader, build_tokenizer
+
+PATH_TO_TOKENIZER = "/workspace/mm/tokenizer.model"
+BATCH_SIZE = 16
+BATCH_NUMBER = 4
+
+
+def main():
```

Comment:
maybe we can make this as a unit test? WDYT?

Reply:
I would add as a unit test some checks of shapes & types on the DP axis rather than this script that just checks the amount of padding in each batch